idb/postgres/postgres.go

```diff
@@ -1251,30 +1286,27 @@ ON CONFLICT (addr, assetid) DO UPDATE SET amount = account_asset.amount + EXCLUD
 	if !any {
 		db.log.Debugf("empty round %d", round)
 	}
-	var importState importState
-	staterow := tx.QueryRow(`SELECT v FROM metastate WHERE k = 'state'`)
-	var stateJSONStr string
-	err = staterow.Scan(&stateJSONStr)
+
+	maxRoundAccounted, err := db.getMaxRoundAccounted(tx)
```

Comment:
Need to use `getImportState` here. If we were to add another value to `importState` the logic here would clear it out each time we update accounting.

Reply:
fair